cargo-spatial/src/main.rs

```diff
@@ -0,0 +1,36 @@
+use cargo_spatial::{codegen, config::Config, local, opt::*};
+use log::*;
+use simplelog::*;
+use structopt::StructOpt;
+
+fn main() {
+    let opt = Opt::from_args();
+
+    // Initialize the logger.
+    let verbosity = if opt.verbose {
+        LevelFilter::Trace
+    } else {
+        LevelFilter::Warn
+    };
+    SimpleLogger::init(verbosity, Default::default()).expect("Failed to setup logger");
+
+    // Perform the operation selected by the user.
+    match &opt.command {
+        Command::Codegen => {
+            let config = Config::load().expect("Failed to load config");
```

Comment:
`"Failed to load configuration"` -> `"Failed to load configuration in current working directory"`